model/item.cc

```diff
@@ -279,6 +279,14 @@ namespace minsky
     cairo_stroke(cairo);
   }
   
+  bool Item::attachedToDefiningVar() const
+  {
+    if (ports.size()>1)
```

Comment:
guard clause unnecessary - the following loop with zero iterations is equivalent.

Reply:
Thanks, I've removed it.